src/main/java/games/strategy/engine/lobby/server/db/BannedMacController.java

```diff
@@ -36,21 +36,15 @@ public void addBannedMac(final String mac, final Instant banTill) {
     }
     Timestamp banTillTs = null;
     if (banTill != null) {
-      banTillTs = new Timestamp(banTill.toEpochMilli());
+      banTillTs = Timestamp.from(banTill);
     }
     try (final Connection con = Database.getPostgresConnection();
-        final PreparedStatement ps = con.prepareStatement("insert into banned_macs (mac, ban_till) values (?, ?)")) {
+        final PreparedStatement ps = con.prepareStatement("insert into banned_macs (mac, ban_till) values (?, ?) on conflict do update")) {
```

Comment:
In this case we don't want to do nothing (what is the original behaviour), but update the entry instead.
At least this is what the javadocs claim this method does.